torchvision/prototype/models/efficientnet.py

```diff
@@ -80,6 +80,7 @@ class EfficientNet_B0_Weights(WeightsEnum):
             **_COMMON_META,
             "num_params": 5288548,
             "size": (224, 224),
+            "min_size": (1, 1),
```

Comment:
Nit: Since the same value is shared across all variants, it's worth moving this inside `_COMMON_META`. Then we can remove it from all other places.